docs/100-development/110-general/113-branch-structure.md

```diff
@@ -25,3 +25,23 @@ Example:
 If you want you can add the issue number to the branch name, e.g. `feat/new-login-system-#123` or `fix/login-error-#456`.
 
 <!-- TODO: merge info from CONTRIBUTING.md -->
+
+## Commits
```

Comment:
I would move the commits section to another file, because "branch-structure" should be for branch structure. Each new file is a new section in the resulting docs, so we should split separate information into separate files